src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -1211,4 +1244,14 @@ private boolean needNumFilteredGenotypes(){
                 maxFractionFilteredGenotypes != MAX_FRACTION_FILTERED_GENOTYPES_DEFAULT_VALUE ||
                 minFractionFilteredGenotypes != MIN_FRACTION_FILTERED_GENOTYPES_DEFAULT_VALUE;
     }
+
+    /**
+     * Need the number of no-call genotyped samples?
+     *
+     * @return true if any of the filtered genotype samples arguments is used (not the default value), false otherwise
+     */
+    private boolean needNoCallGenotypes(){
+        return maxNOCALLnumber != MAX_NOCALL_NUMBER_DEFAULT_VALUE ||
```

Comment:
hmm, these are doubles so it's awkward to use a null (need to use a Double) and we use such defaults everywhere. I'll leave as is. Open a separate ticket for all such defaults if wanted